src/devTools.js

```diff
@@ -138,6 +143,7 @@ function liftReducer(reducer, initialState) {
         currentStateIndex++;
       }
       stagedActions = [...stagedActions, liftedAction.action];
+      timestamps = [...timestamps, Date.now()];
       break;
```

Comment:
Aaargh, I lost track of what you mean :-) Can you try explaining again?

My suggestion was to record the time in `liftAction` because that corresponds precisely to when it fired in the app.